packages/labs/virtualizer/src/test/scenarios/element-and-directive-parity.test.ts

```diff
@@ -24,29 +26,52 @@ abstract class TestElement extends LitElement {
 
   @property({type: Array, attribute: false})
   public items: Array<number> = [];
+
+  @property({type: Array, attribute: false})
```

Comment:
`type` is only used for attribute converters (it would be more accurately named `typeHint`), so if a property has `attribute: false` it doesn't need a type at all. For test elements, it's academic though and you could just leave all properties off. The converter would only kick in if reflecting to an attribute (which is off) or setting from an attribute, which you're not doing.